swedish-oidc-profile.md

```diff
@@ -349,50 +347,56 @@ Relying Parties MUST follow the requirements in section [3.1.3.7] of \[[OpenID.C
-them. 
+An OpenID Provider compliant with this profile MUST NOT release any identity claims in the ID token, or via the UserInfo endpoint, if they have not been explicitly requested via `scope` and/or `claims` request parameters, or by a policy known, and accepted, by the involved parties. 
 
-Section 5.4 of \[[OpenID.Core](#openid-core)\] states:
+The above requirement does not include the mandatory `sub` claim, and claims that do not reveal identity
+information about the user, for example, transaction identifiers or claims holding information about
+the authentication process.
 
-> The Claims requested by the `profile`, `email`, `address`, and `phone` scope values are returned from the UserInfo Endpoint, as described in Section 5.3.2, when a `response_type` value is used that results in an Access Token being issued. However, when no Access Token is issued (which is the case for the `response_type` value `id_token`), the resulting Claims are returned in the ID Token. 
+Furthermore, an OpenID Provider MUST NOT release any claims to a Relying Party that has not been authorized to receive them. How this authorization is handled and managed is out of scope for this
+profile. 
 
-This means that, unless explicitly requested via the `claims` request parameter, identity
-claims<sup>1</sup> are delivered from the UserInfo endpoint. 
+An OpenID Provider compliant with this profile MUST adhere to the following claim release requirements:
 
-OpenID Providers compliant with this profile MUST adhere to the above statements, unless governed
-by an overriding policy that states otherwise.
-  
-"Authentication Information Claims" as defined in section 2.3 of \[[OIDC.Sweden.Attr](#attr-spec)\] are
-claims describing an authentication event, and are not regarded as user identity claims. Therefore, any
-of these claims SHOULD be delivered in the ID token and SHOULD NOT be delivered from the UserInfo
-endpoint.
+- If a `claims` request parameter is included in the authentication request for a specific claim,
+the claim is delivered according to the `claims` parameter contents.
+
+- If a `scope` request parameter value is included in the authentication request, and this scope
+definition has specific claims delivery requirements (i.e., whether the claims belonging to the
+scope should be delivered in ID Token or via the UserInfo endpoint), the claims are delivered
+according to these scope requirements.
 
-> \[1\]: Apart from the mandatory `sub` claim that also can be seen as an identity attribute. 
+- If an overriding policy (to this profile) is effective and this policy has particular requirements
+concerning claims delivery, this policy is applied. 
+
```

Comment:
Nit picking:

The text below has requirement language (MUST) but not the statements above. This could be solved by the following rewording:

"(i.e., whether the claims belonging to the scope **are** delivered in ID Token or via the UserInfo endpoint), the claims **MUST** be delivered according to these scope requirements.

Reply:
Ths bullet shuld not be here due to that the intension will be misread.